k2/csrc/ragged_ops.h

```diff
@@ -292,6 +292,18 @@ std::vector<RaggedShape> UnsqueezeParallel(int32_t num_srcs, RaggedShape **src,
 */
 RaggedShape RemoveAxis(RaggedShape &src, int32_t axis);
 
+/*
+  Returns a `sub-shape` of `src` consisting of one of its RaggedShapeLayer
+  elements, i.e. one of the levels of its shape.  This returned shape
+  will have NumAxes() == 2, but it is the minimal case of a RaggedShape.
+
+    @param [in] src   Source RaggedShape
+    @param [in] layer  Layer that is desired, from 0 .. src.NumAxes() - 2.
+                      View this as an index into its Layers() vector.
+ */
+RaggedShape GetLayer(RaggedShape &src, int32_t layer);
```

Comment:
not sure, but if it's possible, it's better if we can change the parameter to `const RaggedShape& src` in case we'd call this function with lvalue.

Reply:
ok..